sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/implementation/context/AzureTokenCredentialAutoConfiguration.java

```diff
@@ -63,7 +63,7 @@ public class AzureTokenCredentialAutoConfiguration extends AzureServiceConfigura
     @ConditionalOnMissingBean(name = DEFAULT_TOKEN_CREDENTIAL_BEAN_NAME)
     @Bean(name = DEFAULT_TOKEN_CREDENTIAL_BEAN_NAME)
     @Order
-    TokenCredential tokenCredential(DefaultAzureCredentialBuilderFactory factory,
+    TokenCredential springCloudAzureDefaultCredential(DefaultAzureCredentialBuilderFactory factory,
```

Comment:
why do we need to change this?

Reply:
It's convenient for searching, because its name is the updated one, necessary?

Reply:
I don't think so, because if someone not familiar with this name (like me), I won't be able to search it anyways.